pkg/drivers/vmware/vmrun.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ * Copyright 2017 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package vmware
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/docker/machine/libmachine/log"
+)
+
+var (
+	vmrunbin    = setVmwareCmd("vmrun")
+	vdiskmanbin = setVmwareCmd("vmware-vdiskmanager")
+)
+
+var (
+	ErrMachineExist    = errors.New("machine already exists")
+	ErrMachineNotExist = errors.New("machine does not exist")
+	ErrVMRUNNotFound   = errors.New("VMRUN not found")
+)
+
+func init() {
+	// vmrun with nogui on VMware Fusion through at least 8.0.1 doesn't work right
+	// if the umask is set to not allow world-readable permissions
+	SetUmask()
+}
+
+func vmrun(args ...string) (string, string, error) {
+	cmd := exec.Command(vmrunbin, args...)
+	if os.Getenv("MACHINE_DEBUG") != "" {
```

Comment:
This is repeated over and over. Can the check be abstracted into a simple `func isMachineDebugEnabled() bool { }` ?